README.md

```diff
@@ -1 +1,16 @@
 # dataverse-bridge-plugin-lib
+The _bridge-plugin-lib_ mainly consists of interfaces that are used by the bridge-service in order to identify and treat all plugins the same way. 
+The plugins need to implement the interfaces.
+
```

Comment:
This is not so much information, please link to the Service repo so we can read more.